lints/base.go

```diff
@@ -12,7 +12,7 @@ var (
 
 type lintReportUpdater func(*LintReport, ResultStruct)
 
-const ZLintVersion = 2
+const ZLintVersion = 3
```

Comment:
Based on a conversation w/ Zakir, he mentioned doing this for new lints. Maybe it's something we need to think about more closely.